lib/sync-settings.coffee

```diff
@@ -44,7 +44,7 @@ SyncSettings =
   serialize: ->
 
   getGistId: ->
-    gistId = atom.config.get 'sync-settings.gistId'
+    gistId = process.env.GIST_ID or atom.config.get 'sync-settings.gistId'
```

Comment:
Please give preference to the setting rather than the environment variable. Similar to #374.

Reply:
I applied the suggested change in daeeb6552b1c6dcf6996c0621a8f288a87472909.